src/firefly/js/ui/WorkspaceViewer.jsx

```diff
@@ -22,8 +23,8 @@ import {showInfoPopup, INFO_POPUP} from './PopupUtil.jsx';
 import {HelpIcon} from './HelpIcon.jsx';
 import {dispatchAddActionWatcher} from '../core/MasterSaga.js';
 
-const HMargin = 15;
-const VMargin = 15;
+const HMargin = '1rem';
+const VMargin = '1rem';
```

Comment:
For margin and padding, it's better to use JoyUI spacing.  You can specify spacing using Stack component or in `sx` prop.  e.g. `<Stack spacing={1}>`  or `sx={{m:1}}`